packages/react-instantsearch/examples/autocomplete/README.md

```diff
@@ -0,0 +1,8 @@
+This example shows how to build an autocomplete with react-instantsearch
+
+To start the example: 
+
+* npm install
+* npm start
+
+Read more about react-instantsearch [in our documentation](https://community.algolia.com/instantsearch.js/react/).
```

Comment:
Well I think that I would like to have a bit more context before diving in the example itself. But sure, comments in the code would help too! Do you plan on creating a dedicated guide?